docs/source/algorithms/MatchSpectra-v1.rst

```diff
@@ -0,0 +1,72 @@
+
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+Calculate the factor(s) that best match the individual spectra to a
+reference spectrum using `Gauss-Markov process
+<https://en.wikipedia.org/wiki/Gauss%E2%80%93Markov_process>`_. This
+is the same algorithm used by the "blend" step of `PDFgetN
+<http://pdfgetn.sourceforge.net/>`_. The data undergoes the linear
+transformation
+
+.. math::
+
+   y_{reference} = scale * y_{spectrum} + offset
```

Comment:
should be just `scale * y_{spectrum} + offset` and add "such that the distance between :math:`y_{spectrum}` and the transformed spectrum is minimized."